src/Chapter15/Listing15.19.AnInnerJoinUsingSystem.Linq.Enumerable.Join.cs

```diff
@@ -26,12 +28,16 @@ public static void Main()
                     ));
 
 
-            foreach ((int Id, string Name, string Title, Department Department) item in items)
+            foreach
+                ((int Id, string Name, string Title,
+                    Department Department) item in items)
```

Comment:
I think this can be made into one line?
```suggestion
            foreach ((int Id, string Name, string Title, Department Department) item in items)
```

Reply:
too long